test/unit/math/prim/err/check_finite_test.cpp

```diff
@@ -0,0 +1,128 @@
+#include <stan/math/prim/arr.hpp>
+#include <stan/math/prim/mat.hpp>
+#include <stan/math/prim/scal.hpp>
+#include <gtest/gtest.h>
+#include <limits>
+#include <vector>
+#include <string>
+
+using stan::math::check_finite;
+
+// ---------- check_finite: vector tests ----------
+TEST(ErrorHandlingArr, CheckFinite_Vector) {
+  const char* function = "check_finite";
```

Comment:
[optional] Could the function name be defined only once outside of all tests? If you decide to tackle it, do it for all the tests, I won't repeat the same thing in all files. :)

Reply:
Done.

Reply:
Well, actually I feel we should not do this as this would make creating the so-called jumbo tests (more tests files merged in a single file) difficult.